src/lucky_cli/task.cr

```diff
@@ -60,6 +68,131 @@ abstract class LuckyCli::Task
     end
   end
 
+  macro positional_arg(arg_name, description, required = nil, format = nil, to = nil)
+    {% PARSER_OPTS << arg_name %}
+    @{{ arg_name.id }} : {% if to %}Array(String){% else %}String{% end %} | Nil
```

Comment:
Positional args could be a `String` like `User`, but also `Array(String)` like `["name:String", "email:String"]`.